builds/checkin.yaml

```
@@ -11,14 +11,32 @@ jobs:
 ################################################################################
     displayName: Dunfell
     pool:
-      name: azureiotedge
+      name: "iotedge-1es-hosted-linux"
       demands:
-        - yocto
-    timeoutInMinutes: 360
+      - ImageOverride -equals agent-aziotedge-ubuntu-20.04-msmoby
```

```suggestion
      - ImageOverride -equals agent-aziotedge-ubuntu-20.04-msmoby
      - DiskSizeGiB -equals 150
      - WorkFolder -equals /mnt/storage/_work
```
This will add a 150 GiB data disk to the agent (so we can remove the global setting from the pool), and it will change your work folder to the data disk. That will allow you to remove the "Copy Files" step below. I also wonder if it will resolve user permissions problems? The agent user will be the owner of the work folder, and `poky-container` is supposed to automatically use the permissions of the folder you pass in to the `--workdir` arg.

---

I have a forked branch here to test: 
https://github.com/Azure/meta-iotedge/pull/89